src/analytics/analytics.js

```diff
@@ -375,6 +376,12 @@ function filterEventsInPeriod (es, [startTime, endTime]) {
   return es.filter(e => moment(e.timestamp).isSameOrBefore(endTime) && moment(e.timestamp).isAfter(startTime))
 }
 
+// Takes a bunch of events that have .timestamp field and returns only those that
+// happened in the period specified via (startTime, endTime) or before it.
+function filterEventsUpToAndInPeriod (es, [startTime, endTime]) {
```

Comment:
While we know we are talking about events here, `es` feels cryptic as a variable name. 

```suggestion
function filterEventsUpToAndInPeriod (events, [startTime, endTime]) {
```

Reply:
Yeah, this was a bit too short :D. Fixed!